### PR TITLE
Update URLs from Azure to sonic-net

### PR DESCRIPTION
**Pull request by maipbui**
Signed-off-by: maipbui <maibui@microsoft.com>

### CODE REVIEW

File: doc/mgmt/Management Framework.md [hunk 1]
```diff
@@ -2552,7 +2552,7 @@ Following are the list of Open source tools used in Management framework
 ## 12 Appendix B
 
 Following are the list of Open source libraries used in telemetry container. 
-Always refer to the [Makefile](https://github.com/Azure/sonic-telemetry/blob/master/Makefile) for the sonic-telemetry container for current package list.
```
Comment (qiluo-msft) [inline doc/mgmt/Management Framework.md:2555]: >Azure/sonic-telemetry [](http://example.com/codeflow?start=50&length=21)

This repo is migrated in different way:
Azure/sonic-telemetry -> sonic-net/sonic-gnmi

There are 4 occurrences.

Comment (maipbui) [reply doc/mgmt/Management Framework.md:2555]: Updated.

File: doc/mgmt/Management Framework.md [hunk 2]
```diff
@@ -216,8 +216,8 @@ Management framework makes use of the translation library (Translib) written in
...
-The gNMI support requires the gNMI server which is provided as a part of sonic-telemetry container. We would like to rename this container as the sonic-gnmi container as now it can perform configurations as well through the gNMI server.
-Will introduce a new container sonic-mgmt-common to host the common code that is used both in the mgmt-framework and sonic-telemetry container. This new repo will compile into static libraries that will be used in the other two repos. This way sonic-telemetry repo can be compiled without the mgmt-framework being present in the code base.
+The gNMI support requires the gNMI server which is provided as a part of sonic-gnmi container. We would like to rename this container as the sonic-gnmi container as now it can perform configurations as well through the gNMI server.
+Will introduce a new container sonic-mgmt-common to host the common code that is used both in the mgmt-framework and sonic-gnmi container. This new repo will compile into static libraries that will be used in the other two repos. This way sonic-gnmi repo can be compiled without the mgmt-framework being present in the code base.
```
Comment (qiluo-msft) [inline doc/mgmt/Management Framework.md:221]: Let's focus on fixing URLs in this PR, do not touch plain text.

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
> <br> Refer [HLD document](https://github.com/Azure/sonic-telemetry/pull/77) and below mentioned PR's for more details. 

Thanks! This change make sense.

---
Refers to: doc/SONiC_202106_Release_Notes.md:53 in 1171a52. [](commit_id = 1171a52d62160b6b9e98d13d792a4cfc8acc38d3, deletion_comment = False)

File: doc/sonic-build-system/build-enhancements.md [hunk 3]
```diff
@@ -557,5 +557,5 @@ files/build/versions/
 ## References
 
 - Ref:
-	- https://github.com/Azure/SONiC/blob/master/doc/sonic-build-system/DPKG%20caching%20framework%20.ppt
+	- https://github.com/sonic-net/SONiC/blob/master/doc/sonic-build-system/DPKG%20caching%20framework%20.pptx
```
Comment (qiluo-msft) [inline doc/sonic-build-system/build-enhancements.md:560]: >0.pptx [](http://example.com/codeflow?start=101&length=6)

Thanks for the fix!

**Review by qiluo-msft — DISMISSED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**